Sources/Classes/Full View/ViewController.swift

```diff
@@ -67,6 +67,6 @@ open class ViewController: UIViewController {
     override open func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
         
-        animationView?.spruce.animate(animations, duration: duration, animationType: animationType, sortFunction: sortFunction, prepare: false)
+        animationView?.spruce.animate(animations, duration: duration, animationType: animationType, sortFunction: sortFunction)
```

Comment:
If you accept the above suggestions restoring the "prepare by default" behavior, then this should be:
```suggestion
        animationView?.spruce.animate(animations, duration: duration, animationType: animationType, sortFunction: sortFunction, options: [])
```